Soomla/Assets/WebPlayerTemplates/SoomlaConfig/ios/ios-profile-gamecenter/libSoomlaiOSProfileGameCenter.a

```diff
@@ -0,0 +1 @@
+../../../../../../submodules/ios-profile/build/ios-profile-gamecenter/libSoomlaiOSProfileGameCenter.a
```

Comment:
@EugeneButusov, I have a concern if you use such path, it can be put to the release bundles, but in those bundles, there is no file at this path.